File: Modul-6/Py/PRAK603-2310817110006-Adrian_Bintang_Saputera.py
n1, n2 = map(int, input("Input\n").split())

if n1 != n2:
    print("\nOutput")
    print("Jumlah tidak sama")
else:
    matriks1 = list(map(int, input().split()))
    matriks2 = list(map(int, input().split()))

    print("\nOutput")
    for i in range(n1):
        print(matriks1[i] * matriks2[i], end=" ")
    print()

File: Modul-6/C/PRAK603-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int n1, n2;

  printf("Input\n");
  scanf("%d %d", &n1, &n2);

  if (n1 != n2)
  {
    printf("\nOutput\n");
    printf("Jumlah tidak sama\n");
    return 0;
  }

  int matriks1[n1];
  for (int i = 0; i < n1; i++)
  {
    scanf("%d", &matriks1[i]);
  }

  int matriks2[n2];
  for (int i = 0; i < n2; i++)
  {
    scanf("%d", &matriks2[i]);
  }

  printf("\nOutput\n");
  for (int i = 0; i < n1; i++)
  {
    printf("%d ", matriks1[i] * matriks2[i]);
  }
  printf("\n");

  return 0;
}